### DISCUSSION TITLE
v.2.14.5 RS picker

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#12751] [Help] [answered]
Hello all

Sadly in v.2.14.5 Relatioship picker isn't yet working in bigger lists

![imagem](https://github.com/Budibase/budibase/assets/104722906/77a191f0-5fe7-4f36-9a94-61d94bd6e4c8)

**Comment by melohagan** [comment]
Hey @JCV67 

Is this in a Form block? Also was this working in 2.14.4, or is it that you have just happened to notice it now?

**Reply by JCV67** [reply]
No, it is an hand made form.

**Reply by melohagan** [reply]
And from which version of Budibase have you noticed this?

**Reply by JCV67** [reply]
The app is in v2.10.12 because RS Picker does not work in later versions.

**Reply by melohagan** [reply]
Cheers, were you just updating us that the issue is still persisting since then?

**Reply by JCV67** [reply]
Yes, it is persistent, I reported it more than once, I always try the update using a test copy of the app but it still does not work. The only difference is that now, instead of the picked item I see the word 'null', before it was like this:

https://github.com/Budibase/budibase/assets/104722906/6f70a73d-a44a-4f5c-80db-f2ef94b9549c

**Reply by melohagan** [reply, marked as answer]
Okay thank you for the update! There is an open issue here, we haven't forgotten about it: https://github.com/Budibase/budibase/issues/11478

**Reply by JCV67** [reply]
Thank you @melohagan.
